index.js

```diff
@@ -84,16 +84,7 @@ const psOutputRegex = /^[ \t]*(?<pid>\d+)[ \t]+(?<ppid>\d+)[ \t]+(?<uid>\d+)[ \t
 const nonWindowsSingleCall = async (options = {}) => {
 	const flags = options.all === false ? 'wwxo' : 'awwxo';
 
-	// TODO: Use the promise version of `execFile` when https://github.com/nodejs/node/issues/28244 is fixed.
-	const [psPid, stdout] = await new Promise((resolve, reject) => {
-		const child = childProcess.execFile('ps', [flags, psFields], {maxBuffer: TEN_MEGABYTES}, (error, stdout) => {
-			if (error === null) {
-				resolve([child.pid, stdout]);
-			} else {
-				reject(error);
-			}
-		});
-	});
+	const {stdout, psPid} = await execFile('ps', [flags, psFields], {maxBuffer: TEN_MEGABYTES});
```

Comment:
I don't think this is correct. Where is the `pdPid` property coming from?

Reply:
Can you take a look at my latest change? Should work now